src/CheckoutSdk/HttpResponseMessageExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Checkout
+{
+    /// <summary>
+    /// Extensions for <see cref="HttpResponseMessage"/>.
+    /// </summary>
+    public static class HttpResponseMessageExtensions
+    {
+        static private ISerializer _serializer = new JsonSerializer();
+        static private HttpStatusCode _statusCode;
+        static private HttpResponseHeaders _headers;
+        static private HttpContent _contentInput;
+        static private dynamic _contentOutput = null;
+        static private string _json;
+
+        /// <summary>
+        /// Converts a provided <paramref name="httpResponseMessage"/> into a <see cref="CheckoutHttpResponseMessage{dynamic}"/>.
+        /// </summary>
+        /// <param name="httpResponseMessage">The <see cref="HttpResponseMessage"/> to be transformed.</param>
+        /// <returns>The <see cref="CheckoutHttpResponseMessage{TContent}"/> representation of the <see cref="HttpResponseMessage"/>.</returns>
+        public static async Task<CheckoutHttpResponseMessage<TContent>> ConvertToChekoutHttpResponseMessage<TContent>(this HttpResponseMessage httpResponseMessage)
+        {
+            return await ComposeCheckoutHttpResponseMessage<TContent>(httpResponseMessage, typeof(TContent));
+        }
+
+        /// <summary>
+        /// Converts a provided <paramref name="httpResponseMessage"/> into a <see cref="CheckoutHttpResponseMessage{dynamic}"/>.
+        /// </summary>
+        /// <param name="httpResponseMessage">The <see cref="HttpResponseMessage"/> to be transformed.</param>
+        /// <param name="responseType">The type of the response.</param>
+        /// <returns>The <see cref="CheckoutHttpResponseMessage{dynamic}"/> representation of the <see cref="HttpResponseMessage"/>.</returns>
+        public static async Task<CheckoutHttpResponseMessage<dynamic>> ConvertToChekoutHttpResponseMessage(this HttpResponseMessage httpResponseMessage, Type responseType)
```

Comment:
Typo, should be `ConvertToCheckoutHttpResponseMessage `

Reply:
seems to happen more often here

Reply:
fixed with commit 0fcd798